### DISCUSSION TITLE
glowroot over https

### DISCUSSION THREAD

**Discussion by bxourias** [glowroot/glowroot#927] [General]
is there any information on how to configure the port for https (ssl) and how to add authentication to the glowroot UI ?

**Comment by nowheresly** [comment]
Hi @bxourias 

I would recommend to start by reading this section:
[https://github.com/glowroot/glowroot/wiki/Central-Collector-Installation#ui-over-https](https://github.com/glowroot/glowroot/wiki/Central-Collector-Installation#ui-over-https)
